bitswap/network/ipfs_impl.go

```diff
@@ -28,7 +28,7 @@ import (
 
 var log = logging.Logger("bitswap/network")
 
-var connectTimeout = time.Second * 5
+var connectTimeout = time.Second * 10
```

Comment:
With https://github.com/libp2p/go-libp2p/pull/2907 being the default in [go-libp2p 0.36](https://github.com/ipfs/ipfs-desktop/releases/tag/v0.36.0),  we probably could remove this timeout, but maybe aligning with stream timeout from that PR is safer. 

@aschmahmann @gammazero any concerns if we bump this to 15, to match modern timeouts in go-libp2p?

FYSA in latest go-libp2p, both Dial and NewStream have default timeouts of 15s:
https://github.com/libp2p/go-libp2p/blob/v0.36.0/p2p/net/swarm/swarm.go#L28-L36


```suggestion
var connectTimeout = time.Second * 15
```